File: Demo/Sources/ViewControllers/Dialog/Content/DialogContentViewController.m
//
//  Globus
//

#import "DialogContentViewController.h"

@interface DialogContentViewController ()
@end

@implementation DialogContentViewController

#pragma mark - Actions

- (IBAction)pressedHide:(id)sender {
    [self.glb_dialogViewController dismissWithCompletion:nil];
}

@end

File: Demo/Sources/ViewControllers/Dialog/Content/DialogContentViewController.swift
//
//  Globus
//

import Globus

class DialogContentViewController: GLBViewController {
    
    // MARK - Action
    
    @IBAction internal func pressedHide(_ sender: Any) {
        self.glb_dialogViewController?.dismiss()
    }
    
    // MARK: - GLBNibExtension
    
    public override static func nibName() -> String {
        return "DialogContentViewController-Swift"
    }
    
}
